packages/Manager/src/main/resources/db/migration/R__setup_permissions.sql

```diff
@@ -0,0 +1,20 @@
+GRANT
+  USAGE ON SCHEMA public TO eop_manager_app_user;
+
+ALTER DEFAULT PRIVILEGES
+  IN SCHEMA public
+GRANT
+SELECT
+,
+  INSERT,
+UPDATE
+,
+  DELETE ON TABLES TO eop_manager_app_user;
```

Comment:
Gaa ... this makes me wonder if the SQL formatting that Spotless is doing is worth it ... this is pretty ugly

Reply:
Yeah, the line-breaks and indentation seem a little weird here. 

https://github.com/sql-formatter-org/sql-formatter is an alternative integrated with prettier. The output doesn't seem any better on this file (through their [demo](https://sql-formatter-org.github.io/sql-formatter/)), but there is some configurability.

https://sqlfum.pt produces nicer output, but seems newer/experimental. 

I'd be OK with manual formatting for now, given the low volume of SQL code likely compared with other languages.